modal/sandbox.py

```diff
@@ -158,11 +159,18 @@ async def _load(provider: _Sandbox, resolver: Resolver, _existing_object_id: Opt
 
             sandbox_id = create_resp.sandbox_id
             provider._hydrate(sandbox_id, resolver.client, None)
-            provider._stdout = LogsReader(api_pb2.FILE_DESCRIPTOR_STDOUT, sandbox_id, resolver.client)
-            provider._stderr = LogsReader(api_pb2.FILE_DESCRIPTOR_STDERR, sandbox_id, resolver.client)
 
         return _Sandbox._from_loader(_load, "Sandbox()", deps=_deps)
 
+    def _hydrate_metadata(self, handle_metadata: Optional[Message]):
```

Comment:
could be wrong but i think `Message` is always set now, so you don't have to make it `Optional` and check that it's not `None`

Not sure why it's still `Optional` in the base class. But looking at `_Function` and `_Cls` they both assume it's set

I can fix this separately